fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/query/plan/cascades/rules/AbstractDataAccessRule.java

```diff
@@ -402,8 +402,10 @@ private static Map<PartialMatch, RelationalExpression> distinctMatchToScanMap(@N
      * intersection. For single data scans that will not be used in an intersection we still follow the same
      * two-step approach of separately planning the scan and then computing the compensation and the compensating
      * expression.
+     *
```

Comment:
I am not sure what these changes stem from but can you undo them in this file?

Reply:
Maybe remnants of previous changes and cleanups .. will revert.